digdag-ui/.gitignore

```diff
@@ -2,4 +2,5 @@
 node_modules
 .DS_Store
 npm-debug.log
+npm-debug.log.*
```

Comment:
Files are like `npm-debug.log.564447649`. So *.log doesn't work.
I prefer making it restricted so that we can avoid unexpected missing files.